concent_api/core/payments/bankster.py

```diff
@@ -367,23 +367,7 @@ def settle_overdue_acceptances(
             transaction_type=TransactionType.BATCH,
         )
 
-        # Concent defines time T1 equal to youngest timestamp from list of transactions.
-        if len(list_of_transactions) > 0:
-            youngest_transaction_timestamp = max(
-                ethereum_transaction.closure_time for ethereum_transaction in list_of_transactions
-            )
-
-            assert youngest_transaction_timestamp <= cut_off_time
-
-            # Concent checks if all passed SubtaskResultAccepted messages from subtask_results_accepted_list
-            # have payment_ts < T1
-            if any(
-                youngest_transaction_timestamp > subtask_results_accepted.payment_ts
-                for subtask_results_accepted in acceptances
-            ):
-                raise BanksterTimestampError
-        else:
-            raise BanksterTimestampError
+        validate_list_of_transactions_timestamps(list_of_transactions, cut_off_time, acceptances)
```

Comment:
So what about if `len(list_of_transactions) == 0` ?

Reply:
Nothing, it pass validation and `settle_overdue_acceptances` return `claim_against_requestor`.